modules/fingerprint/tests/testRun/root_tree/file1

```diff
@@ -0,0 +1 @@
+
```

Comment:
These are helping files for the testcases: https://github.com/resin-os/resinhup/blob/master/modules/fingerprint.py#L120